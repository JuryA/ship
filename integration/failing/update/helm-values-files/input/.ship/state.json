{
  "v1": {
    "config": {},
    "releaseName": "jenkins",
    "kustomize": {
      "overlays": {
        "ship": {
          "excludedBases": [
            "/jenkins-test.yaml"
          ],
          "patches": {
            "/jenkins-master-deployment.yaml": "apiVersion: apps/v1beta1\nkind: Deployment\nmetadata:\n  labels:\n    chart: jenkins-0.20.1\n    component: jenkins-jenkins-master\n    heritage: Tiller\n    release: jenkins\n    custom: label\n    key: value\n  name: jenkins\n"
          }
        }
      }
    },
    "upstream": "github.com/helm/charts/tree/78858a2fb07aebe082efb11b8991b60e7fdd5d60/stable/jenkins",
    "metadata": {
      "applicationType": "helm",
      "icon": "https://wiki.jenkins-ci.org/download/attachments/2916393/logo.png",
      "name": "jenkins",
      "releaseNotes": "Added ExtraPorts to open in the master pod (#7759)\n\n* Added ExtraPorts to open in the master pod\n\nSigned-off-by: ecejjar \u003cjesus.javier.arauz@ericsson.com\u003e\n\n* [stable/jenkins] Bumped up chart version\n\nSigned-off-by: ecejjar \u003cjesus.javier.arauz@ericsson.com\u003e",
      "license": {
        "assignee": "",
        "createdAt": "0001-01-01T00:00:00Z",
        "expiresAt": "0001-01-01T00:00:00Z",
        "id": "",
        "type": ""
      },
      "sequence": 0,
      "version": "0.20.1"
    },
    "contentSHA": "c25b3cd58e801f1a642961c860c4162da2c171fa8a8cd33d32012c36631535e9"
  }
}
