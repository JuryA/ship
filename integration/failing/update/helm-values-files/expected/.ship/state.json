{
  "v1": {
    "config": {},
    "releaseName": "jenkins",
    "upstream": "github.com/helm/charts/tree/78858a2fb07aebe082efb11b8991b60e7fdd5d60/stable/jenkins",
    "kustomize": {
      "overlays": {
        "ship": {
          "excludedBases": [
            "/jenkins-test.yaml"
          ]
        }
      }
    },
    "metadata": {
      "applicationType": "helm",
      "icon": "https://wiki.jenkins-ci.org/download/attachments/2916393/logo.png",
      "name": "jenkins",
      "releaseNotes": "Added ExtraPorts to open in the master pod (#7759)\n\n* Added ExtraPorts to open in the master pod\n\nSigned-off-by: ecejjar \u003cjesus.javier.arauz@ericsson.com\u003e\n\n* [stable/jenkins] Bumped up chart version\n\nSigned-off-by: ecejjar \u003cjesus.javier.arauz@ericsson.com\u003e",
      "license": {
        "assignee": "",
        "createdAt": "0001-01-01T00:00:00Z",
        "expiresAt": "0001-01-01T00:00:00Z",
        "id": "",
        "type": ""
      },
      "sequence": 0,
      "version": "0.20.1"
    },
    "contentSHA": "f19d8cd21704fb28480ac2f4b590522cf1686ae3b2bc2c6e7eb71da9b169947a"
  }
}
